applications/plugins/SofaCUDA/CMakeLists.txt

```diff
@@ -236,7 +235,7 @@ endif()
 
 set(README_FILES README.md)
 
-find_package(Sofa.Component.Mass REQUIRED)
+sofa_find_package(Sofa.Component.Mass REQUIRED)
```

Comment:
Could you explain the difference and in which situations use one or the other?

Reply:
@guparan already told me... but I don't remember :)

Reply:
I am not @guparan  but for me it is just a wrapper around find_package(), which creates convenient cmake vars and macros (like SOFACUDA_HAVE_SOFA_COMPONENT_MASS in this case) to be used in the config.h.in.
there is also a small check if the CMake config file has already been parsed (so a little speedup while cmake-configuring)